doc/adr/0008-use-ods-code-as-canonical-url.md

```diff
@@ -0,0 +1,21 @@
+# 8. Use ODS Code as Canonical URL
+
+Date: 2017-09-25
+
+## Status
+
+Accepted
+
+## Context
+
+Profile records are currently identified only by the Choices (PIMS) ID.
+A better option is the ODS Code which is a system wide identifier.
```

Comment:
Suggest adding more context to this statement e.g. `A better option is the ODS Code which is an NHS system wide identifier.`